src/notify/notify.c

```diff
@@ -213,6 +213,15 @@ static int run(int argc, char* argv[]) {
         if (r == 0)
                 return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP),
                                        "No status data could be sent: $NOTIFY_SOCKET was not set");
+
+        /* 5,000,000 µs is 5s */
+        r = sd_notify_barrier(0, 5000000);
```

Comment:
hmm, i think there might be value in not synchronizing here...

i.e. maybe add an option "--no-block" for this? i.e. that we block by default, except if people tell us to?

Also write the timeout as `5 * USEC_PER_SEC`, it's self-explanatory then.